test/udev-test.pl

```diff
@@ -43,52 +43,75 @@
 my @tests = (
         {
                 desc            => "no rules",
-                devpath         => "/devices/pci0000:00/0000:00:1f.2/host0/target0:0:0/0:0:0:0/block/sda",
-                exp_name        => "sda" ,
-                exp_rem_error   => "yes",
+                devices => [
+                        {
+                                devpath         => "/devices/pci0000:00/0000:00:1f.2/host0/target0:0:0/0:0:0:0/block/sda",
+                                exp_name        => "sda" ,
+                                exp_rem_error   => "yes",
+                        },
+                        {
+                                devpath         => "/devices/pci0000:00/0000:00:1f.2/host0/target0:0:0/0:0:0:0/block/sda/sda1",
+                                exp_name        => "sda1" ,
+                                exp_rem_error   => "yes",
```

Comment:
So this one is actually the single (?) case where `devices` has two items. It'd be great to make this a separate commit, so it is not buried in all the other trivial cases where `devices` has one item.